[2022-09-17 02:11:33,172] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2019-01-01T06:00:00+00:00 [queued]>
[2022-09-17 02:11:33,182] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2019-01-01T06:00:00+00:00 [queued]>
[2022-09-17 02:11:33,183] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-09-17 02:11:33,183] {taskinstance.py:1239} INFO - Starting attempt 10 of 10
[2022-09-17 02:11:33,183] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-09-17 02:11:33,256] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): ingest> on 2019-01-01 06:00:00+00:00
[2022-09-17 02:11:33,266] {standard_task_runner.py:52} INFO - Started process 645 to run task
[2022-09-17 02:11:33,276] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDag', 'ingest', 'scheduled__2019-01-01T06:00:00+00:00', '--job-id', '247', '--raw', '--subdir', 'DAGS_FOLDER/ingest_data_dag.py', '--cfg-path', '/tmp/tmpjqnrsi7w', '--error-file', '/tmp/tmpbgcu821b']
[2022-09-17 02:11:33,278] {standard_task_runner.py:77} INFO - Job 247: Subtask ingest
[2022-09-17 02:11:33,373] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDag.ingest scheduled__2019-01-01T06:00:00+00:00 [running]> on host 5d94276fbe2b
[2022-09-17 02:11:33,516] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-09-17 02:11:33,566] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2019-01-01T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-01T06:00:00+00:00
[2022-09-17 02:11:33,567] {logging_mixin.py:109} INFO - root   root   docker_sql_0_pgdatabase_1   5432   ny_taxi   yellow_taxi_2019_01   /opt/***/output_2019-01.parquet
[2022-09-17 02:12:44,796] {local_task_job.py:154} INFO - Task exited with return code Negsignal.SIGKILL
[2022-09-17 02:12:44,823] {taskinstance.py:1277} INFO - Marking task as FAILED. dag_id=LocalIngestionDag, task_id=ingest, execution_date=20190101T060000, start_date=20220917T021133, end_date=20220917T021244
[2022-09-17 02:12:46,041] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-09-17 02:12:46,079] {dagrun.py:530} ERROR - Marking run <DagRun LocalIngestionDag @ 2019-01-01 06:00:00+00:00: scheduled__2019-01-01T06:00:00+00:00, externally triggered: False> failed
[2022-09-17 02:12:46,080] {dagrun.py:605} INFO - DagRun Finished: dag_id=LocalIngestionDag, execution_date=2019-01-01 06:00:00+00:00, run_id=scheduled__2019-01-01T06:00:00+00:00, run_start_date=2022-09-17 02:10:43.351323+00:00, run_end_date=2022-09-17 02:12:46.080302+00:00, run_duration=122.728979, state=failed, external_trigger=False, run_type=scheduled, data_interval_start=2019-01-01 06:00:00+00:00, data_interval_end=2019-02-01 06:00:00+00:00, dag_hash=6059d761a110d1304e9bdc6239d9f30f
[2022-09-17 03:27:26,734] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2019-01-01T06:00:00+00:00 [queued]>
[2022-09-17 03:27:26,745] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2019-01-01T06:00:00+00:00 [queued]>
[2022-09-17 03:27:26,745] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-09-17 03:27:26,745] {taskinstance.py:1239} INFO - Starting attempt 10 of 10
[2022-09-17 03:27:26,746] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-09-17 03:27:26,783] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): ingest> on 2019-01-01 06:00:00+00:00
[2022-09-17 03:27:26,789] {standard_task_runner.py:52} INFO - Started process 7491 to run task
[2022-09-17 03:27:26,796] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDag', 'ingest', 'scheduled__2019-01-01T06:00:00+00:00', '--job-id', '266', '--raw', '--subdir', 'DAGS_FOLDER/ingest_data_dag.py', '--cfg-path', '/tmp/tmpmbpfh2o2', '--error-file', '/tmp/tmp0901pw8q']
[2022-09-17 03:27:26,798] {standard_task_runner.py:77} INFO - Job 266: Subtask ingest
[2022-09-17 03:27:26,904] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDag.ingest scheduled__2019-01-01T06:00:00+00:00 [running]> on host 5d94276fbe2b
[2022-09-17 03:27:27,004] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-09-17 03:27:27,057] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2019-01-01T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-01T06:00:00+00:00
[2022-09-17 03:27:27,058] {logging_mixin.py:109} INFO - root   root   docker_sql_0_pgdatabase_1   5432   ny_taxi   yellow_taxi_2019_01   /opt/***/output_2019-01.parquet
[2022-09-17 03:28:40,826] {local_task_job.py:154} INFO - Task exited with return code Negsignal.SIGKILL
